train_supervised.py

```diff
@@ -30,6 +31,7 @@
 # normalize all pixel values to be in [-1, 1] and add Gaussian noise with mean zero, variance gaussian_noise_var
 transform = transforms.Compose(
     [transforms.ToTensor(),
+     # [0.4914, 0.4822, 0.4465], [0.2488, 0.2453, 0.2633]
```

Comment:
Not sure where the (0.5, 0.5, 0.5) mean and std dev came from, but the values in the comments here are the empirical mean and std computed over the training data (per channel).  That's technically what you should be using in Normalize, though it won't make much of a difference at all to performance.